output/output.go

```diff
@@ -112,6 +117,18 @@ func (o Output) String() string {
 	return b.String()
 }
 
+func (o Output) applyTemplate() string {
+	t, err := template.New("template").Parse(o.Metadata.Config.Output.Template)
+	if err != nil {
+		return err.Error()
+	}
+	var b strings.Builder
+	if err := t.Execute(&b, o); err != nil {
+		return err.Error()
+	}
+	return b.String()
+}
+
```

Comment:
As `Output.String()` is called only when we export the final result, an error inside the template given by the user would only happen here.

Thus everything when fine but there's nothing to show because the template is broken.

Could we use the default if an error occurs here ?
```go
func (o Output) applyTemplate() (string, error)
```

Reply:
Ideally we'd manage this via the config validation as suggested in the issue:
> Optional: perform syntax validation before the run in `config.Global.Validate`

But it requires some re-architecturing because it implies to import `output` in `config` 😕 

Until then, the fallback you suggest is a great idea, let's do this!